### DISCUSSION TITLE
[A-L] (12 Gennaio 2022) Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#357] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152746048-230b8a89-6e19-46ef-bad7-9f7d827689c1.png)

**Comment by matypist** [comment]
_Simile ma non uguale a #146_

**Comment by i0nut02** [comment]
<img width="636" alt="Schermata 2022-02-07 alle 10 05 41" src="https://user-images.githubusercontent.com/99051485/152757738-c06ee645-25e1-4e1b-be64-9ddb3ceea36e.png">

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/210138814-92445b9d-4ede-49bf-afa1-e340b33c9c81.png)

**Comment by Elia-Belli** [comment]
**Notazione**: $P_i$ indica se la proposizione $i$ è vera, $A_i$ indica se il pulsante $i$ innesca l'autodistruzione.
- **(1)**: "Questo pulsante non innesca l'autodistruzione": $P_1 = \lnot A_1$ = "il primo pulsante non innesca l'autodistruzione";
- **(2)**: "Questo pulsante non innesca l'autodistruzione": $P_2 =\lnot A_2$ = "il secondo pulsante non innesca l'autodistruzione";
- **(3)**: "Il primo pulsante innesca l'autodistruzione": implicitamente ci dice anche che se stesso (il terzo pulsante) non innesca l'autodistruzione $P_3 = \lnot A_3 \land A_1$ = "il primo pulsante innesca l'autodistruzione e il terzo no";
- **Condizioni**: solo 1 tasto ha la scritta corretta (nella tabella prendiamo solo i casi in cui uan $P$ è vera) e solo 1 innesca l'autodistruzione (prendiamo solo i casi in cui solo una $A$ è vera).

$A_1$ $A_2$ $A_3$ $\space$ $P_1$ $P_2$ $P_3$
0  $\space \space$ 0 $\space \space$  1 $\space \space$ 1 $\space \space$ 1 $\space \space$ 0
0  $\space \space$ 1  $\space \space$ 0 $\space \space$ 1 $\space \space$ 0 $\space \space$ 0 
1  $\space \space$ 0  $\space \space$ 0 $\space \space$ 0 $\space \space$ 1 $\space \space$ 1

Quindi l'innesco è il **pulsante 2**.

Spiegazione della tabella:
- Se $A_3 = 1$, ovvero "il pulsante 3 è l'innesco", $P_1$ e $P_2$ sono entrambe vere (impossibile per ipotesi), quindi il pulsante 3 non fa esplodere.
- Se $A_2 = 1$, ovvero "il pulsante 2 è l'innesco", $P_2$ e $P_3$ sono entrambe false, quindi tutte le condizioni sono soddisfatte, solo il pulsante 2 fa esplodere e solo $P_1$ è vera.
- Se $A_1 = 1$, ovvero "il pulsante 1 è l'innesco", $P_2$ e $P_3$ sono entrambe vere (impossibile per ipotesi), quindi il pulsante 1 non fa esplodere.
